openscap_daemon/cve_scanner/cve_scanner.py

```diff
@@ -277,7 +277,16 @@ def signal_handler(self, signal, frame):
         sys.exit(0)
 
     def search_containers(self, image, cids, output):
-        f = Scan(image, cids, output, self.ac)
+        try:
+            f = Scan(image, cids, output, self.ac)
+        except Exception as e:
+            # We don't know all types of docker/atomic exception, so we catch
+            # all these exceptions to avoid daemon freezing
+            self.failed_scan = str(error)
```

Comment:
Even with this fixed it's not clear to me why we are doing `self.failed_scan =` and logging an error in another part of the code. Could you please explain the differences?